.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# IDEs
+.idea/
```

Comment:
JFYI: you can always add files to `.git/info/exclude` without being noticed

Reply:
Ok, thx for the hint!